:memo: Rework CDN docs (#2746)
platformsh/platformsh-docs: docs/src/domains/cdn/managed-fastly.md

aliases:
 - "/golive/steps/fastly.html"
---

Instead of starting your own Fastly subscription and managing your CDN yourself,
you can take advantage of a Fastly CDN provided by Platform.sh.
For example, Dedicated projects include a managed Fastly CDN by default.
These CDNs are exclusively set up and managed by Platform.sh.
To modify any settings for a managed Fastly CDN,
[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).

### Domain control validation
 
When you request for a new domain to be added to your Fastly service,
Platform.sh support provides you with a [`CNAME` record](../../domains/steps/dns.md) for domain control validation.
To add this `CNAME` record to your domain settings,
contact your DNS provider.

### Transport Layer Security (TLS) certificates
 
By default, Enterprise and Elite plans include two [TLS certificates](../../other/glossary.md#transport-layer-security-tls),
an apex and a wildcard one.
This allows for encryption of all traffic between your users and your app.
 
If you use a Fastly CDN provided by Platform.sh, 
you can provide your own third-party TLS certificates for an additional fee.

> [Transfer each certificate](../../development/file-transfer.md),
> its unencrypted private key, 
> and the intermediate certificate to a non-web accessible [mount](../../create-apps/app-reference.md#mounts),
> on an environment that only Platform.sh support and trusted users can access.

Reviewer: :pick: To help with the sentence length warning at the top. :shrug: 
```suggestion
If you don't have one, set up a [mount](../../create-apps/app-reference.md#mounts)
that isn't accessible to the web.
Use an environment with access limited to Platform.sh support and trusted users.
[Transfer](../../development/file-transfer.md) each certificate, its unencrypted private key, 
and the intermediate certificate to the mount.
```

---
title: "Managed Fastly CDN"
sidebarTitle: "Managed Fastly CDN"
weight: 2
description: Bring your content closer to users with a Fastly CDN fully managed by Platform.sh.
tier:
  - Elite
  - Enterprise
---

Instead of starting your own Fastly subscription and [managing your CDN yourself](./fastly.md),
you can take advantage of a Fastly CDN provided by Platform.sh.
For example, Dedicated projects include a managed Fastly CDN by default.
These CDNs are exclusively set up and managed by Platform.sh.

To modify any settings for a managed Fastly CDN,
[open a support ticket](https://console.platform.sh/-/users/~/tickets/open).

> For information on how to subscribe to a managed Fastly CDN,

Reviewer: :pick: To increase directness and potentially avoid confusion with subscriptions.
```suggestion
To add a managed Fastly CDN to your project,
```

> [contact our sales team](https://platform.sh/contact/).

Reviewer: :clipboard: To avoid the first person.
```suggestion
[contact sales](https://platform.sh/contact/).
```
